packages/sheets-data-validation/src/views/date-dropdown/index.tsx

```diff
@@ -128,7 +128,7 @@ export function DateDropdown(props: IDropdownComponentProps) {
                 }}
             />
             <div className={styles.dvDateDropdownBtns}>
-                <Button size="small" type="primary" onClick={handleSave} disabled={!localDate}>
+                <Button size="small" type="primary" onClick={handleSave} disabled={!date.isValid()}>
                     确定
```

Comment:
It seems like here might be i18n issues here.

Reply:
get